telco-examples/edge-clusters/airgap/README.md

```diff
@@ -70,18 +70,18 @@ The following steps are required to prepare the airgap artifacts:
   - You need to create a list with the Helm charts required for the edge cluster. For example, for telco scenarios, you can use the following list:
     ``` 
     $ cat > edge-release-helm-oci-artifacts.txt <<EOF
-    edge/sriov-network-operator-chart:302.0.0+up1.4.0
-    edge/sriov-crd-chart:302.0.0+up1.4.0
+    edge/sriov-network-operator-chart:303.0.2+up1.5.0
+    edge/sriov-crd-chart:303.0.2+up1.5.0
```

Comment:
Shouldn't we remove the -charts suffix here as well?

Reply:
yep good catch!! I changed from